### ISSUE TITLE
How to handle the same process name？

### ISSUE THREAD

**Issue by xincili** [ncabatoff/process-exporter#242]
If there are multiple processes with the same name on a machine, how does the component distinguish between the different processes?

**Comment by lethargosapatheia** [comment]
By PID I suppose?

**Comment by xincli** [comment]
No, how to monitor for processes like containers where the pid changes frequently, but there are multiple processes with the same process name?
